test/connector/tcp/mssql/mssql_connector_test.go

```diff
@@ -37,17 +38,22 @@ func RunTests(t *testing.T, queryExec dbQueryExecutor) {
 	})
 
 	t.Run("Cannot connect directly to MSSQL", func(t *testing.T) {
+		// Set Host and Port to $DB_HOST_TLS and $DB_PORT environment
```

Comment:
nit: for some reason this little chunk looks messy, but that could just just be me

Reply:
Roger that... What's going on here is that I need to read 2 environment variables ($DB_HOST_TLS and $DB_PORT), and merge those into the default secretless DB config. One alternative would have been to read these env variables directly via os, and merge in with defaults. But I figured there was already a testutil.NewDbConfigFromEnv() function to read config from env, so I just used that function, and merged results. The other gotcha was that $DB_PORT env is a string, so it needed an Atoi() call with an error check.